File: src/main/java/com/manifestors/shinrai/client/module/modules/visuals/NoHurtCam.java
package com.manifestors.shinrai.client.module.modules.visuals;

import com.manifestors.shinrai.client.module.Module;
import com.manifestors.shinrai.client.module.ModuleCategory;
import com.manifestors.shinrai.client.module.annotations.ModuleData;

@ModuleData(
        name = "NoHurtCam",
        description = "Disables camera shake effect when taking damage.",
        category = ModuleCategory.VISUALS
)
public class NoHurtCam extends Module {

}

File: src/main/kotlin/com/manifestors/shinrai/client/module/modules/visuals/NoHurtCam.kt
package com.manifestors.shinrai.client.module.modules.visuals

import com.manifestors.shinrai.client.module.Category
import com.manifestors.shinrai.client.module.Module
import org.lwjgl.glfw.GLFW

class NoHurtCam : Module(
    name = "NoHurtCam",
    description = "Disables hurt cam effect when you are taking damage.",
    category = Category.VISUALS,
    key = GLFW.GLFW_KEY_B
)